settings.gradle.kts

```diff
@@ -38,4 +38,6 @@ pluginManagement {
         gradlePluginPortal()
         mavenCentral()
     }
-}
\ No newline at end of file
+}
+
+rootProject.name = "kermit-root"
```

Comment:
Is this needed? What is this added for?

Reply:
Yeah, there is a bug in BCV related to https://github.com/Kotlin/binary-compatibility-validator/issues/261.

Reply:
Ooops, it still failed on CI, see https://github.com/touchlab/Kermit/actions/runs/11709073063/job/32612290197.

May have to revert this change until the issue has been fixed on BCV side.